content/docs/use-cases/shared-development-server.md

```diff
@@ -55,14 +55,23 @@ your <abbr>project</abbr>:
 $ dvc cache dir /home/shared/dvc-cache
 ```
 
-And tell DVC to set group permissions on newly created or downloaded cache
-files:
+External cache configuration:
 
 ```dvc
 $ dvc config cache.shared group
+$ dvc config cache.type "symlink,hardlink,copy"
 ```
 
-> See `dvc cache dir` and `dvc config cache` for more information.
+Above, we first tell DVC to set group permissions on new cache files. Then we
+enable soft and hard [file links](/doc/user-guide/large-dataset-optimization)
+(you can pick only one or the other) to avoid having copies from the external
```

Comment:
`you can pick only one or the other` sounds like you should pick in the command itself

btw, hardlinks most likely won't work also :)

so, it's only about symlinks usually

Reply:
Makes sense. Left symlinks only and improved the note that links to the other types.